.github/workflows/auto-release-branch-e2e.yaml

```
@@ -0,0 +1,108 @@
+name: Auto Build e2e-runner for New Release Branch
+
+on:
+  create:
```

Does this need to be qualified so it doesn't fire for ALL topic branch creation?

```suggestion
  create:
    branches:
      - 'release-1.**'
```